nixos/tests/minidlna.nix

```diff
@@ -0,0 +1,39 @@
+import ./make-test.nix ({ pkgs, ... }: {
+  name = "minidlna";
+
+  nodes = {
+    server =
+      { ... }:
+      {
+        imports = [ ../modules/profiles/minimal.nix ];
+        networking.firewall.allowedTCPPorts = [ 8200 ];
```

Comment:
Perhaps the module could use an "openFirewall" option?

According to https://help.ubuntu.com/community/MiniDLNA, we might want to open UDP port 1900 too.

Reply:
Personally, i use it solely with the tcp port 8200 and it run fine so far ;)
Also, I checked my configuration, and indeed i opened that port myself.

Then again, that was the current behavior so i kept it.
The proposed patch is about allowing more configuration, not behavior change.

Also, i don't really know what's preventing this to be merged already so i'd rather not change the scope just yet ;)

Reply:
Ok, fair enough. (Although it wouldn't be behavioural change since openFirewall options (should) default to false in nixpgks.)